web/src/components/FiltersForm.res

```diff
@@ -153,6 +154,7 @@ module Filters = {
       ->URLSearchParams.get(name)
       ->Js.Nullable.bind((. value) => {
         set(_ => filter->Filter.validate(value) ? value : "")
+        // TODO(fbo) disable related fields when project parameter is set
```

Comment:
Could the api extend the query when those fields are provided with the project def?

Reply:
It was not the intend, and it seems messy ihmo to do that. I prefer to disable related field. It is already done when you select a project from the list. That's not a big issue, so it can be done in another PR.